circuit_knitting_toolbox/entanglement_forging/cholesky_decomposition.py

```diff
@@ -388,16 +354,10 @@ def _get_fermionic_ops_with_cholesky(
 
     if halve_transformed_h2:
         h2 /= 2  # type: ignore
-    h1_int = OneBodyElectronicIntegrals(basis=ElectronicBasis.SO, matrices=h1)
-    h2_int = TwoBodyElectronicIntegrals(basis=ElectronicBasis.SO, matrices=h2)
-    int_property = IntegralProperty("fer_op", [h1_int, h2_int])
```

Comment:
This integral property was set up with integrals in the SO basis, which is no longer available. [ElectronicBasis now only supports AO and MO](https://github.com/Qiskit/qiskit-nature/blob/main/qiskit_nature/second_q/problems/electronic_basis.py).

The problem I'm having is that the new code below is yielding ops of N qubits, rather than N/2 qubits, and I'm guessing it is because we are not in the spin basis.

Any thoughts @mrossinek?